03-task-manager/starter/errors/custom-error.js

```diff
@@ -0,0 +1,13 @@
+
+class CustomAPIError extends Error {
+    constructor (message, statusCode) {
+        super(message)
+        this.statusCode = statusCode
+    }
+}
+
+const createCustomError = (message, statusCode) => {
+    return CustomAPIError(message, statusCode)
```

Comment:
You forgot the new to instantiate the CustomAPIError.
Ex: new CustomAPIError(message, statusCode)